nvmw.bat

```diff
@@ -75,27 +81,44 @@ exit /b 0
 :install
 setlocal
 
+set NODE_TYPE=node

```

Comment:
@fengmk2 If you can do it, I think keep compatibility is good for current users.